streaming.md

```diff
@@ -10,18 +10,18 @@ In the case of Matroska, there are mostly 2 different kinds of stream: file acce
 
 # File Access
 
-File access can simply be reading a file located on your computer, but also accessing it from an HTTP (web) server or CIFS (windows share) server. All these protocol are usually safe from reading errors and seeking in the stream is possible. On other hand when the file is stored far or on a slow server, seeking can be an expensive operation and should be avoided. That's why we set a [few guidelines]({{site.baseurl}}/index.html) that, when followed, help reduce the number of seeking for regular playback and also have the playback start quickly without a lot of data needed to read first (like the Cues (index), Attachments or Meta Seek of all the Clusters).
+File access can simply be reading a file located on your computer, but also accessing it from an HTTP (web) server or CIFS (windows share) server. All these protocol are usually safe from reading errors and seeking in the stream is possible. On other hand when the file is stored far or on a slow server, seeking can be an expensive operation and SHOULD be avoided. That's why we set a [few guidelines]({{site.baseurl}}/index.html) that, when followed, help reduce the number of seeking for regular playback and also have the playback start quickly without a lot of data needed to read first (like the Cues (index), Attachments or Meta Seek of all the Clusters).
```

Comment:
"all these protocols" should be plural; "is stored far or…" → "is stored far away or…"